packages/@tinacms/api-git/src/router.ts

```diff
@@ -80,7 +94,13 @@ export function router(config: GitRouterConfig = {}) {
       console.log(fileAbsolutePath)
     }
     try {
-      writeFile(fileAbsolutePath, req.body.content)
+      const fileIsInRepo = checkFilePathIsInGitRepo(
+        fileAbsolutePath,
+        REPO_ABSOLUTE_PATH
```

Comment:
I just realized this should be:

```suggestion
        CONTENT_ABSOLUTE_PATH
```

If the site is in a subdirectory (i.e. if you have multiple sites in one repo) you shouldn't be able to edit content outside of that site.

Reply:
It would be good to update the variable names (i.e. `fileIsInRepo`) accordingly.

Reply:
what you suggest as name? `fileIsInSite` ?